recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -26,87 +29,92 @@ class AndreasbuhrCppCoroConan(ConanFile):
         "fPIC": True,
     }
 
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _minimum_compilers_version(self):
         return {
             "Visual Studio": "15",
+            "msvc": "191",
             "gcc": "10",
             "clang": "8",
             "apple-clang": "10",
         }
 
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        
         # We can't simply check for C++20, because clang and MSVC support the coroutine TS despite not having labeled (__cplusplus macro) C++20 support
```

Comment:
What if we check C++20 in a branch for gcc etc, and else check C++17 for clang & msvc?

Reply:
We're already checking 17 across the board, but I can see if GCC with C++17 fails.

Reply:
GCC 10+ in C++17 mode works, so checking for 20 would potentially exclude some valid configurations. So actually the comment seems to apply to all compilers, they all work in C++17 mode. If anything, I think mentioning the C++ standard is confusing, really all we're checking for is the coroutines TS, as the error message says. The C++17 check above is sufficient for the C++ standard check.

Reply:
So perhaps we can say something like "In addition to C++17, cppcoro requires Coroutine TS support".

Reply:
@RubenRBS Can you please give this another look? I think this is the last actionable comment (or if you wanted to test some clang configurations).

Reply:
@Ahajha Our maintainer, Rubén, is on vacation, but will return on next Monday 😄